docs/tutorials/migrations/homebrew.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Homebrew
+description: Common questions and solutions for using Rust with Flox
+---
+
+# Homebrew migration guide
+Flox is a virtual environment manager and package manager all in one. Using Flox, you can create virtual environments containing packages from the Flox Catalog.
+
+Flox can replace Homebrew entirely, or they can be used together.
+
+This guide explains how to introduce Flox into environments where Homebrew is currently being used, either as a replacement or an addition. It introduces new concepts and proposes a basic procedure for mapping packages.
+
+## Why you might want to migrate
+Homebrew does a great job, and has been loved as the “missing package manager” for a generation of macOS users. But there are a few reasons you might consider moving to Flox:
+
+* **You need a cross-platform package manager.** Flox works on both macOS and Linux - x86 and ARM - allowing you to define a set of packages that works the same everywhere.
+* **You need a virtual package manager** Flox allows you to create as many environments as you need, each with a different set of packages.
+* **You need reproducible environments.** Flox environments are defined in a TOML manifest that can be easily shared, or managed alongside code.
+* **You need a broader set of software.** Homebrew has a sizeable collection of packages, but the Flox Catalog is based on Nixpkgs, the largest collection of packages in the world.
+* **You need older versions of software.** The Flox Catalog contains historical versions of each of its packages - in many cases, going back decades.
+
+## Install Flox
+Download and install Flox following the [installation instructions]().
+
+## Find and install your first package
+Migrating to Flox is a straightforward process of installing Flox packages for each of your Homebrew formulae. This section walks you through the process for identifying the set of Homebrew formulae you currently have installed, searching for the corresponding packages in the Flox Catalog, and installing them.
+
+As a Homebrew user, you will find several of the Flox subcommands familiar:
+
+* `search` is used to find available packages
+* `install` is used to install packages
+* `uninstall` is used to remove packages
+
+### Show top-level formulae in Homebrew
+First, identify the list of Homebrew packages you have installed.
+
+We recommend using `brew leaves` for this, so you can easily differentiate between the formulae you installed explicitly versus those that were installed as dependencies. The `leaves` subcommand will show the formulae that were directly installed.
+
+For example:
+
+```
+% brew leaves
+boost
+gh
+imagemagick
+mysql
+openssh
+```
+
+Then, search Flox for one of the formulae you have installed with Homebrew. In this case, for example, you could choose `boost`:
+
+```
+% flox search boost
```

Comment:
This example is odd in that there are many packages at concurent versions on each revision of nixpkgs (so versioning happens via the name and via the version). 

This is totally a nonblocking suggestion, but I wonder if another package would be easier to illustrate the points with.